content/en/platform/corda/5.0/key-concepts/_index.md

```diff
@@ -10,5 +10,76 @@ menu:
+These rules are expressed as {{< tooltip >}}smart contracts{{< /tooltip >}} that are intrinsically linked with the chosen representation of the global state.
+Every participant in the network has a view of the shared global state of data (the extent of that view depends on its visibility restrictions), which is updated in real-time as changes occur.
+This ensures that the data is accurate, transparent, and tamper-evident.
+
+Blockchain is one popular type of DLT that is commonly used for cryptocurrencies such as Bitcoin, but there are many other types of DLT besides blockchain.
+
+## What is Corda’s take on DLT?
+Corda is a distributed application platform for the creation and operation of decentralized applications, which are written using a rich Java API.
+These applications are created using standard Java development tools and packaged using Corda tooling.
+Once packaged, applications are deployed onto Corda which executes both applications and ledger update workflows on behalf of organizational entities.
+
+There are some approaches that differentiate Corda from other DLTs. Identity (along with associated permissions) and privacy are especially important in order to serve the needs of highly regulated financial institutions. The following summarizes what Corda does differently:
+
+* **Identity** - Knowing who you are transacting with and knowing that their identity was attested to a given level of assurance is important for establishing trust.
+This is especially the case for regulated industries that need to perform some level of KYC (know your customer).
+Often this is because of anti-money laundering (AML) legislation that requires you to know who you are transacting with, but it can also be for other reasons such as avoiding the legal consequences of engaging with an entity on a sanction list.
+Identity validation is handled when onboarding a participant identity to a business network.
+Many other DLT and blockchain systems offer anonymity as a feature, especially for cases where true censor-resistant mutation of some global state is a requirement.
+However, for use cases where trust is absolutely critical, Corda layers its platform with the concept of granting permissions to identities.
+These permissions can be granted during onboarding of identities and it is simply a case of attesting that supplied credentials prove that an entity is who they claim to be.
+Corda does not mandate certain levels of attestation, but rather each network is able to set their own rules. Without permissions, access to the state of a network and other members of that network is impossible.
+* **Privacy** - People who want to communicate with each other on Corda can install the same set of distributed applications, known as {{< tooltip >}}CorDapps{{< /tooltip >}}.
+These define the parameters of their interactions and exchange information and assets; all within the bounds of what the network operator has permitted around rules for visibility of states.
```

Comment:
I think the parts that overlap are all small pieces here and there rather than chunks so the overhead of snippets might outweigh the gain, especially as I wouldn't expect this type of content to evolve over time.